snuba/datasets/processors/search_issues_processor.py

```diff
@@ -215,6 +212,10 @@ def process_insert_v1(
                 )
             client_timestamp = _client_timestamp
 
+        retention_days = enforce_retention(
```

Comment:
does this need to be part of the migration PR? can this be merged independently?